vite.config.ts

```diff
@@ -23,9 +23,13 @@ export default defineConfig(({ mode }) => {
     },
     plugins: [react()],
     envPrefix: 'NB_',
-    // Excluding the Auth0 library from the bundle to avoid issues with
-    // Cypress component tests. TODO: understand why this is necessary
+
     optimizeDeps: {
+      // Including MUI and emotion here for local
+      // Cypress component testing to work
+      include: ['@emotion/react', '@emotion/styled', '@mui/material/Tooltip'],
```

Comment:
Curious, were the tests failing cause these were missing?

Reply:
yes, I actually wanted to ask you this: if you run cypress component tests locally (i.e. not GH runner) and without first building the tool (i.e. unlike GH runner by using `npm run dev` instead of `build`), do the component tests pass for me?

Reply:
I'm not so sure how the component tests function with vite. Best way to be sure is to just try and see what happens.

Reply:
I tried to replicate my locally failing component tests by removing the `include` array above and was not able to. So I'm no longer sure what actually failed.

The error message I was getting was: `styled_default is not a function`, specifically when running the `GetDataDialog` test. 

I'll remove the line from the config for now